DuckDuckGo.xcodeproj/project.pbxproj

```diff
@@ -6517,6 +6516,7 @@
 				F1D43AF92B99C1D300BAB743 /* BareBonesBrowserKit */,
 				9F8FE9482BAE50E50071E372 /* Lottie */,
 				9F96F73A2C9144D5009E45D5 /* Onboarding */,
+				1E5918462CA422A7008ED2B3 /* Navigation */,
```

Comment:
Required for `WKNavigationAction`, `HTTPURLResponse` and other navigation related extensions.